config/checkstyle_resources_suppressions.xml

```diff
@@ -167,4 +167,906 @@
+    files="[\\/]InputVariableDeclarationUsageDistanceCheck\.java"/>
+  <suppress checks="FileLength"
+    files="[\\/]InputOperatorWrap\.java"/>
+  <suppress checks="FileLength"
+    files="[\\/]InputMethodParamPad\.java"/>
+  <suppress checks="FileLength"
+    files="[\\/]InputModifierOrder\.java"/>
+  <suppress checks="FileLength"
+    files="[\\/]InputIndentationCorrectReturnAndParameter\.java"/>
+  <suppress checks="FileLength"
+    files="[\\/]InputIndentationCorrectWhileDoWhileAndParameter\.java"/>
+  <suppress checks="FileLength"
+    files="[\\/]InputIndentationCorrectFieldAndParameter\.java"/>
+  <suppress checks="FileLength"
+    files="[\\/]InputParenPad\.java"/>
+  <suppress checks="FileLength"
+    files="[\\/]InputWhitespaceAroundBasic\.java"/>
+  <suppress checks="FileLength"
+    files="[\\/]InputLineLength\.java"/>
+  <suppress checks="FileLength"
+    files="[\\/]InputNoFinalizeExtend\.java"/>
+  <suppress checks="FileLength"
+  files="[\\/]ExpectedAstTreeStringPrinterFullOfSinglelineComments\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]ExpectedAstTreeStringPrinterFullOfBlockComments\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]expectedVoidElementsAst\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]expectedRpHtmlTagAst\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]expectedHtmlVoidElementEmbedAst\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]expectedHtmlVoidElementTrackAst\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]expectedRbHtmlTagAst\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]expectedOptgroupHtmlTagAst\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]expectedNonTightRtHtmlTagAst\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]expectedNonTightOptgroupHtmlTagAst\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]expectedHtmlVoidElementWbrAst\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]expectedNestingWithSingletonElementAst\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]expectedNonTightRpHtmlTagAst\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]expectedRtcHtmlTagAst\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]expectedParagraphAst\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]expectedHtmlVoidElementKeygenAst\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]expectedRtHtmlTagAst\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]expectedHtmlVoidElementSourceAst\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]expectedNonTightRbHtmlTagAst\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]expectedNonTightRtcHtmlTagAst\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]expectedAllStandardJavadocTagsAst\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]expectedUppercaseInPackageNameAst\.txt"/>
+  <suppress checks="FileLength"
+    files="[\\/]expectedSpaceBeforeDescriptionInBlockJavadocTagsAst\.txt"/>
```

Comment:
I would think we would not make violations on expected files. It is direct output of the input after going through the Checkstyle process. If the input file is flagged to be trimmed, then it will already affect the expected file. But if we are flagging only the expected file, then it may need extra scrutiny if we can even fix the issue.

@romani ping

Reply:
> If the input file is flagged to be trimmed, then it will already affect the expected file.

I think that we would have to suppress expected files irrespective of whether the input file for the Checkstyle process is trimmed or not. The input file can be within the limit but the printed AST can exceed the length of 120.
Example - 
https://github.com/checkstyle/checkstyle/blob/master/src/test/resources/com/puppycrawl/tools/checkstyle/asttreestringprinter/ExpectedAstTreeStringPrinterFullOfSinglelineComments.txt
The [input](https://github.com/checkstyle/checkstyle/blob/master/src/test/resources/com/puppycrawl/tools/checkstyle/asttreestringprinter/InputAstTreeStringPrinterFullOfSinglelineComments.java) file is under the limit but the output file is not.

Reply:
It depends on the type of output, but it may be a possibility we can trim the input file more so the output is within the size too. This is why I pinged @romani to confirm.

Reply:
Yes, we need to exclude txt files, we need to validate only java files.

Reply:
For extra clarification after talking to @romani , we will only do Java files as part of this issue since this is the main chunk of our inputs. We will handle non-Java input files in a new issue. We will exclude "Expected" or "Output" files for now.

Reply:
We can just use `fileExtensions` property of FileLengthCheck until #11178

Reply:
https://checkstyle.org/config_sizes.html#FileLength_Properties yes, we can.

Reply:
Done